docs/server/installation.rst

```diff
@@ -388,55 +232,23 @@ software. To run it, just issue:
 
    (env) $ pootle start
 
-
```

Comment:
well almost all - and looking in other files it mostly single line after blocks - unless its the start of a new section